locale/fr/about/governance.md

```diff
@@ -0,0 +1,146 @@
+
+Pour la liste des Collaborateurs actuels, voir le [README.md]
+(https://github.com/nodejs/node/blob/master/README.md#current-project-team-members) du projet.
+
+Un guide des Collaborateurs est maintenu sur 
+[COLLABORATOR_GUIDE.md](https://github.com/nodejs/node/blob/master/COLLABORATOR_GUIDE.md).
+
+## Siéger au TSC
+
+Les sièges au TSC ne sont pas limités dans le temps. Le TSC n'a pas
+de taille fixe. Cependant, l'objectif recherché est entre 6 et 12 personnes, 
+afin d'assurer une couverture adéquate des différents domaines d'expertise, 
+tout en conservant une capacité de prise de décision efficace.
+
+Il n'y a aucun prérequis ou qualifications nécessaires au-delà de
+ces règles pour siéger au TSC.
+
+Le TSC peut ajouter des membres additionnels au TSC par une motion
+standard du TSC.
+
+Un membre du TSC peut être retiré du TSC par démission volontaire, ou
+par une motion standard du TSC.
+
+Les changements de modalités de participation au TSC devront être mises
+au programme et peuvent être ajoutés comme n'importe quel autre
+objet de programme (voir "Réunions du TSC" plus bas).
+
+Pas plus d'1/3 des membres du TSC ne pourra être affilié au même
+employeur. Si le retrait ou la démission d'un membre du TSC, ou le 
+changement d'employeur d'un membre du TSC, créé une situation dans
+laquelle plus d'1/3 des membres du TSC partage le même employeur,
+la situation devra immédiatement être corrigée par la démission ou le
+retrait d'un ou plusieurs membres du TSC affiliés avec le ou les 
+employeurs sur-représentés.
+
+## Réunions du TSC
+
+Le TSC se réunit de manière hebdomadaire sur Google Hangout On Air.
+La réunion est animée par un modérateur désigné et approuvé par le TSC.
+Chaque réunion est publiée sur Youtube.
+
+Sont ajoutés au programme les objets qui sont considérés contentieux
+ou sont des modifications de gouvernance, de politique de contribution, de
+participation au TSC, ou concernant le processus de release.
+
+L'intention du programme n'est pas d'approuver ou relire tous les 
+correctifs. Ceci devra être fait de manière continue sur GitHub
+et géré par l'ensemble des Collaborateurs.
+
+N'importe quel membre de la communauté ou contributeur peut demander
+à ajouter un objet au programme de la prochaine réunion en ouvrant un
+ticket sur GitHub. N'importe quel Collaborateur, membre du TSC ou le
+modérateur peut ajouter des objets au programme en ajoutant l'étiquette
+***tsc-agenda*** au ticket.
+
+Avant chaque réunion du TSC, le modérateur partagera le Programme
+avec les membres du TSC. Les membres du TSC pourront ajouter n'importe
+quel objet de leur choix au programme au début de chaque réunion. Ni le
+modérateur ni le TSC ne peuvent retirer ou poser de veto sur les objets.
+
+Le TSC peut inviter des personnes ou des représentants de certains projets
+à participer sans pouvoir de vote. Ces invitations sont actuellement:
+
+* Un représentant du projet [build](https://github.com/node-forward/build)
```

Comment:
this link should be nodejs/build nope ?

Reply:
Didn't touch the link. @nodejs/website can someone answer? Do we need to change the link? On all pages?

Reply:
It redirects to https://github.com/nodejs/build but yes it should be updated. It is probably better to do it in a dedicated PR though.